content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/spawning.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Spawning Autonomous Player Entities
+description: A reference for spawning and registering network player entities
+linktitle: Spawning Players
```

Comment:
Spawning Networked Players?

Reply:
I think Spawning Players is fine considering the context of where this topic is in the docs. This is the title that appear in the ToC. Shorter linkTitles are better so they don't span lines.